katcp/testutils.py

```diff
@@ -28,7 +28,7 @@
 import tornado.testing
 from _thread import get_ident
 from tornado.concurrent import Future as tornado_Future
-from future.utils import native_str_to_bytes
+from future.utils import native_str_to_bytes, bytes_to_native_str  # noqa: E402
```

Comment:
E402 is "Module level import not at top of file", so I wonder why this was necessary.

Reply:
Any time we add an import to the Python 2 Python 3 files that have `standard_library.install_aliases() ` above the other imports we run into this issue and I have to silence it by adding that `#noqa: E402` or disabling the linting. 
See the then jenkins job below that is failing. 
http://ci.camlab.kat.ac.za/job/katcp-multibranch/job/user%252Ftockards%252FP2M-31%252Fstatus-to-str/2/execution/node/22/log/

The other alternative is to ignore this within the `setup.cfg`

Reply:
Oh dear.  I wonder if it is because we are only trying to lint the diff, and the diff excludes the `standard_library.install_aliases()  # noqa: E402`?
Anyway, I think it would be better to disable this one globally, rather than having almost random `#noqa: E402` comments sprinkled throughout the codebase.